Tests/DiffableDataSourceTests/DiffableTests.swift

```diff
@@ -1,5 +1,5 @@
 import XCTest
-@testable import Diffable
+@testable import DiffableDataSource
 
 final class DiffableTests: XCTestCase {
     func testExample() {
```

Comment:
We really should have some tests here. 😀

Reply:
Hello Anthony! It was my idea all along. I bumped into some issues with testing (turns out it's better to use animated false on the tests bc the content gets available right away). 🙈

Here's the pull : https://github.com/Doist/doist-apple-diffable-datasource/pull/7